StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdateCardViewController.swift

```diff
@@ -123,11 +134,29 @@ final class UpdateCardViewController: UIViewController {
         return cardBrandDropDown
     }()
 
+    private lazy var expiryDateElement: TextFieldElement = {
+        let expiryDate = CardExpiryDate(month: paymentMethod.card?.expMonth ?? 0, year: paymentMethod.card?.expYear ?? 0)
+        let expiryDateElement = TextFieldElement.ExpiryDateConfiguration(defaultValue: expiryDate.displayString, isEditable: false).makeElement(theme: appearance.asElementsTheme)
+        return expiryDateElement
+
+    }()
+
+    private lazy var cvcElement: TextFieldElement = {
+        let cardBrandProvider = { [weak self] in
+            self?.paymentMethod.card?.brand ?? .unknown
+        }
```

Comment:
Let's use `paymentMethod.card?.card.preferredDisplayBrand` to better support CBC cards.